src/js/events/__tests__/MetronomeClient-test.ts

```diff
@@ -338,6 +337,7 @@ describe("MetronomeClient", () => {
     it(
       "sends 2 requests if schedule is present",
       observe(() => {
+        mockRequest.mockReturnValue(of({}));
```

Comment:
why do we need more of those?

Reply:
I initially had add these to each test, then added it to the `beforeEach` and missed removing them from the tests.

These extra ones can be removed.